app/models/storage_location.rb

```diff
@@ -126,7 +130,10 @@ def self.import_inventory(filename, org, loc)
       adjustment.line_items
                 .create(quantity: row[0].to_i, item_id: current_org.items.find_by(name: row[1]))
     end
-    adjustment.storage_location.increase_inventory(adjustment)
+
+    increasing_adjustment, _decreasing_adjustment = adjustment.split_difference
+
```

Comment:
I don't recommend using adjustment.split_difference.  We've recently discovered that it has the side effect of changing the quantities on the line items attached to the adjustment to be positive.